src/SarifTests/SarifUtilitiesTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.CodeAnalysis.Sarif;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.Sarif.Tests
+{
+    [TestClass()]
+    public class SarifUtilitiesTests
+    {
+        private const string V1_0_0 = "1.0.0";
+        private const string V1_0_0_BETA_5 = "1.0.0-beta.5";
```

Comment:
You don't need to provide these private constants. `ConvertToSchemaUri` takes as its argument a member of the `SarifVersion` enumeration. See below for how to do this.